package-lock.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "@cfn-modules/alb-listener",
```

Comment:
is this file needed?

Reply:
@michaelwittig As we are adding "dependencies" to some of the modules, I thought about adding a package-lock.json to all modules for consistency. Do you agree?

Reply:
ok